backend/src/test/java/de/neuefische/cgnjava222/zoomanagement/zoo/trucks/TruckControllerTest.java

```diff
@@ -0,0 +1,29 @@
+package de.neuefische.cgnjava222.zoomanagement.zoo.trucks;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class TruckControllerTest {
```

Comment:
Super, das Ihr Integration-Test und Unit-Test habt. Dieser Unit-Test kann ja dann den Service in Zukunft testen.